### PR TITLE
Adding failed cluster deletion steps and known issue for 1.2

### PR DESCRIPTION
**Pull request by lparis**
Applies to 1.2 and 1.3/master branches. Out for review. Thanks.

### CODE REVIEW

File: delete-cluster.html.md.erb [hunk 1]
```diff
@@ -3,11 +3,147 @@ title: Deleting Clusters
...
+1. Using the PKS ClI, run the following command to list the Kubernetes clusters deployed by PKS. 
+  <pre class="terminal">$ pks clusters</pre>
+
+1. Using the PKS ClI, run the following command to get the Kubernetes cluster UUID for the target cluster. Replace `{CLUSTERNAME}` with the name of the cluster you want to delete.
+  <pre class="terminal">$ pks cluster {CLUSTERNAME}</pre>
+
+The corresponding BOSH deployment ID includes the `service-instance_` prefix followed by the cluster UUID, for example: `service-instance_cdaee466-539b-4d9c-bd00-b02d766023ea`.
+
+1. Using the BOSH CLI, run the following command to get the deployment ID of the PKS cluster to delete.
+  <pre class="terminal">$ bosh -d "service-instance_$(pks cluster {CLUSTERNAME} --json | jq -cr '.uuid')"</pre>
```
Comment (maplain) [inline delete-cluster.html.md.erb]: could you please replace `bosh -d` here with an `echo`, eg:
```
kubo@jumper:~$ echo "service-instance_$(pks cluster k8s2 --json | jq -cr '.uuid')"
service-instance_cdaee466-539b-4d9c-bd00-b02d766023ea
```

File: delete-cluster.html.md.erb [hunk 2]
```diff
@@ -3,11 +3,147 @@ title: Deleting Clusters
...
+
+##<a id='delete-failed-cluster'></a>Delete Failed Cluster
+
+If the [pks create-cluster](./create-cluster.html) operation results in a failed Kubernetes cluster state, complete the following steps to delete that cluster.
+
+1. <%= partial 'login-pks-api' %>
+
+1. Set the BOSH environment. 
+
+Gather credential and IP address information for your BOSH Director, SSH into the Ops Manager VM, and use BOSH CLI v2+ to log in to the BOSH Director from the Ops Manager VM. There are various methods to do this, two of which are provided below. For more information, see [Advanced Troubleshooting with the BOSH CLI](https://docs.pivotal.io/pivotalcf/customizing/trouble-advanced.html).
```
Comment (Alphasite) [inline delete-cluster.html.md.erb]: `log in to the BOSH Director` confuses me a little, i misread it as ssh into the bosh director.

File: delete-cluster.html.md.erb [hunk 3]
```diff
@@ -3,11 +3,147 @@ title: Deleting Clusters
...
+
+Option 1: Set command line alias that contains the BOSH credential and host. 
+
+```
+alias bosh="BOSH_CLIENT=ops_manager BOSH_CLIENT_SECRET={BOSH-CLIENT-SECRET} BOSH_CA_CERT={CA-CERT-PATH} BOSH_ENVIRONMENT={BOSH-IP-ADDRESS} bosh "
```
Comment (Alphasite) [inline delete-cluster.html.md.erb]: You might be able to simplify this, for the opsman use case, by fetching the pre-populated string from opsman UI, as the required certs, etc are already installed on the opsman vm.

File: delete-cluster.html.md.erb [hunk 4]
```diff
@@ -3,11 +3,147 @@ title: Deleting Clusters
...
+
+1. Using the PKS ClI, run the following command to get the Kubernetes cluster UUID for the target cluster. Replace `{CLUSTERNAME}` with the name of the cluster you want to delete.
+  <pre class="terminal">$ pks cluster {CLUSTERNAME}</pre>
+
+The corresponding BOSH deployment ID includes the `service-instance_` prefix followed by the cluster UUID, for example: `service-instance_cdaee466-539b-4d9c-bd00-b02d766023ea`.
+
+1. Using the BOSH CLI, run the following command to get the deployment ID of the PKS cluster to delete.
+  <pre class="terminal">$ bosh -d "service-instance_$(pks cluster {CLUSTERNAME} --json | jq -cr '.uuid')"</pre>
+
+<p class="note"><strong>Note</strong>: The above command assumes you have installed the <a href="https://stedolan.github.io/jq/">JQ command-line JSON processor</a> to format the JSON response. To install JQ on Linux, run the following command: `sudo apt-get install jq`.</p>
```
Comment (Alphasite) [inline delete-cluster.html.md.erb]: I'm not sure I would advise people to install random CLIs on the opsman VM, especially as the upgrade procedure for opsman, would blow these away and require them to be reinstalled.

Comment (Alphasite) [reply delete-cluster.html.md.erb]: Although I do get the value in it, as it simplifies things, so if you feel that it is OK, then its only a suggestion, not a definite no.

File: delete-cluster.html.md.erb (hunk 1 above)
Comment (lparis) [reply delete-cluster.html.md.erb]: done

File: delete-cluster.html.md.erb (hunk 2 above)
Comment (lparis) [reply delete-cluster.html.md.erb]: done

File: delete-cluster.html.md.erb (hunk 3 above)
Comment (lparis) [reply delete-cluster.html.md.erb]: good idea for an enhancement, thanks, but for now sticking with what works

File: delete-cluster.html.md.erb (hunk 4 above)
Comment (lparis) [reply delete-cluster.html.md.erb]: I think it's ok.

Comment (lparis) [reply delete-cluster.html.md.erb]: But I made mention that if you don't want to do it, you can use `bosh deployments` instead.

**Comment by lparis** [comment]
Done with review. Ready for merge and deployment. thanks

**Comment by vikafed** [comment]
Hi @skelly-pivotal @bstick12. Could you take a look at this PR? Is it OK if merge it?

File: delete-cluster.html.md.erb [hunk 5]
```diff
@@ -3,11 +3,156 @@ title: Deleting Clusters
...
+
+##<a id='delete-failed-cluster'></a>Delete Failed Cluster
+
+If the [pks create-cluster](./create-cluster.html) operation results in a failed Kubernetes cluster state, complete the following steps to delete that cluster.
+
+1. <%= partial 'login-pks-api' %>
+
+1. Set the BOSH environment. 
+
+Gather credential and IP address information for your BOSH Director, SSH into the Ops Manager VM, and run the BOSH CLI to access BOSH Director from the Ops Manager VM. There are various methods to do this, two of which are provided below. For more information, see [Advanced Troubleshooting with the BOSH CLI](https://docs.pivotal.io/pivotalcf/customizing/trouble-advanced.html).
```
Comment (bstick12) [inline delete-cluster.html.md.erb]: I think we should point at the `Bosh Commandline Credentials` in the director credentials tab to get the BOSH command line details.

File: delete-cluster.html.md.erb [hunk 6]
```diff
@@ -3,11 +3,156 @@ title: Deleting Clusters
...
+- BOSH_CA_CERT is the path of the BOSH CA certificate.
+- BOSH_ENVIRONMENT is the IP address of the BOSH Director VM available in the **Status** tab of the BOSH Director tile.
+
+For example:
+
+```
+alias bosh="BOSH_CLIENT=ops_manager BOSH_CLIENT_SECRET=Q336epOEhEzI_8bwgqzGKnO1_Jn9OT2o BOSH_CA_CERT=/var/tempest/workspaces/default/root_ca_certificate BOSH_ENVIRONMENT=30.0.0.2 bosh "
+```
+
+Option 2: Create a `bosh_env.sh` file and source it.
```
Comment (bstick12) [inline delete-cluster.html.md.erb]: I think one option is enough. SSH to Ops Manager box and export the value of the 'Bosh Commandline Credentials'

File: delete-cluster.html.md.erb [hunk 7]
```diff
@@ -3,11 +3,156 @@ title: Deleting Clusters
...
+worker/8720482e-38c7-475f-8199-e7c7a3d6a145  running        az-2  30.0.3.6  vm-b1794d82-e6da-4b2d-b8f2-476bebbdd5b1  medium.disk  true
+worker/a74935ab-21cf-4a96-8aa4-38e7050a4aa9  running        az-3  30.0.3.7  vm-3f5caff1-2706-428d-af4e-485cc74934a9  medium.disk  true
+worker/f2276b8e-af08-4de3-ba60-7b6dc8bd910c  running        az-1  30.0.3.5  vm-4eae862e-b4c7-4aac-8e5e-61960904f41e  medium.disk  true
+
+6 vms
+
+Succeeded
+
+
+$ bosh -d  "service-instance_$(pks cluster k8s2 --json | jq -cr '.uuid')" delete-deployment
```
Comment (bstick12) [inline delete-cluster.html.md.erb]: You might need a `--force` here depending on the route cause of the failure

**Pull request merged**